tests/UI/campaigns/data/faker/cartRule.js

```diff
@@ -28,7 +28,7 @@ module.exports = class CartRule {
     this.quantityPerUser = cartRuleToCreate.quantityPerUser || 1;
 
     // Actions
-    this.freeShipping = cartRuleToCreate.freeShipping === undefined ? 'on' : cartRuleToCreate.freeShipping;
+    this.freeShipping = cartRuleToCreate.freeShipping === undefined ? false : cartRuleToCreate.freeShipping;
```

Comment:
How did this work until now ?

Reply:
When `freeShipping = 'on'`, tests created all cart rules with freeShipping, but it did not affect the total price since we choose free carrier in FO anyway.